bower.json

```diff
@@ -12,6 +12,8 @@
     "jquery": "^1.11.1",
     "loader.js": "ember-cli/loader.js#3.2.0",
     "qunit": "~1.17.1",
-    "bootstrap-daterangepicker": "~1.3.20"
+    "bootstrap-daterangepicker": "~1.3.20",
+    "moment": ">= 2.8.0",
+    "moment-timezone": ">= 0.1.0"
```

Comment:
yes, @josemarluedke it has been added by the `ember-moment` lets not break the inner dependencies.